modules/assets/src/pages/assets-details.vue

```diff
@@ -72,28 +76,55 @@ export default defineComponent({
     },
 
     options: {
-      type: Object,
+      type: Object as PropType<{
+        editableAsset: Image;
+        product: ProductDetails;
```

Comment:
Do we need to pass whole product here? We use only images field later. Should image data editor know anything about product fields except images?